tests/Doctrine/Tests/ODM/PHPCR/Mapping/AbstractMappingDriverTest.php

```diff
@@ -396,7 +412,7 @@ public function testLoadChildrenMapping()
      */
     public function testChildrenMapping(ClassMetadata $class)
     {
-        $this->assertInternalType('array', $class->childrenMappings);
+        $this->assertIsArray($class->childrenMappings);
         $this->assertCount(2, $class->childrenMappings);
         $this->assertArrayHasKey('all', $class->mappings);
         $this->assertArrayNotHasKey('filter', $class->mappings['all']);
```

Comment:
I was checking the tests locally to help move this PR forward, but the failing test here is actually a bug introduces some time ago. 

```diff
- $this->assertFalse(isset($class->mappings['all']['filter']));
+ $this->assertArrayNotHasKey('filter', $class->mappings['all']);
```

But the filter value in the array is `null` in this case. So the previous isset check would work but the current check fails. This needs to be changed to `$this->assertNull($class->mappings['all']['filter']);`

See: https://github.com/doctrine/phpcr-odm/commit/3f23224b00b07097fb5da5e62f2b5bcb5b0fe8be#diff-87c636fa83a2393c03d9d0cbc6c2b33c90cf07e9c0ce08f8c990872df4588f84R395-R402

Reply:
thanks, that explains it. and goes to show that merging without the ci working is dangerous.